src/main/common/color.h

```diff
@@ -21,10 +21,12 @@
 typedef enum {
     RGB_RED = 0,
     RGB_GREEN,
-    RGB_BLUE
+    RGB_BLUE,
+    RGB_WHITE,
```

Comment:
I think values like `RGB_BLUE` are perfectly clear. It means it's blue out of a list that contains values red, green and blue. Although as you point out, it should now be RGBW.

`CC_RED` is less descriptive, it could mean `{255, 0, 0}` for instance.